integration_test/cases/migrator.exs

```diff
@@ -81,6 +86,9 @@ defmodule Ecto.Integration.MigratorTest do
 
       assert count_entries() == 1
       assert [50] = run(TestRepo, path, :up, to: 50, log: false)
+
+      purge migrations
```

Comment:
Something is odd because we don't get those warnings in Postgres. We are
very careful to only use new names (hence the numbers). Thoughts?



-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*